libs/Tree/Directory.php

```diff
@@ -54,10 +54,47 @@ public function getIndexPage()
             return $this->children[$index_key];
         }
 
+        /*
+          If the inherit_index flag is set, then we seek child content
+         */
+        if(
+          !empty($this->getConfig()['live']['inherit_index'])
+          && $first_page = $this->seekFirstPage()
+          ){
+          return $first_page;
+        }
+
         return null;
     }
 
     /**
+     * Seek the first available page from descendants
+     * @return Content|null
+     */
+    public function seekFirstPage(){
```

Comment:
Actually, the more I think about this method, the more I think it is actually the same as `getFirstPage` but that allows to get the first index page.